frontend/src/locales/en-US.json

```diff
@@ -2673,8 +2673,8 @@
   "custom-approval": {
     "self": "Custom Approval",
     "security-rule": {
```

Comment:
It's better to change the key as well. `security-rule` -> `risk-rule`

Reply:
Done